src/components/Config/Config.tsx

```diff
@@ -120,68 +146,148 @@ function Config() {
     }
   }, [checkConnection, wallet]);
 
+  const ArbitrumNetworkConnect = useCallback(async () => {
+    const ArbitrumTestChainId = '0x66eeb'; // arbitrum test net
```

Comment:
camelCase this.
This should also be extracted into `constants.ts`. In general always avoid using these 'magic values' inline. They should be declared in one global place and referenced everywhere else. You can do another enum `ChainId` (PascalCase is ok for enums). Also, please declare these values in decimal first and then you can do another enum like `ChainIdHex` and calculate the value via `ethers.utils.bigNumberify(val).toHexString()` so that you only have to remember to update one place and the rest will be recalculated for you